.pre-commit-hooks.yaml

```diff
@@ -1,10 +1,20 @@
--   id: ruff
-    name: ruff
-    description: "Run 'ruff' for extremely fast Python linting"
-    entry: ruff check --force-exclude
-    language: python
-    'types_or': [python, pyi]
-    args: []
-    require_serial: true
-    additional_dependencies: []
-    minimum_pre_commit_version: '2.9.2'
+- id: ruff
+  name: ruff
+  description: "Run 'ruff' for extremely fast Python linting"
+  entry: ruff check --force-exclude
+  language: python
+  "types_or": [python, pyi]
+  args: []
+  require_serial: true
+  additional_dependencies: []
+  minimum_pre_commit_version: "2.9.2"
+- id: ruff-format
+  name: ruff-format
+  description: "Run 'ruff format' for extremely fast Python formatting"
+  entry: ruff format --check
```

Comment:
Probably want force-exclude here too

Reply:
Ah I wasn't sure it was supported. Great!